go.mod

```diff
@@ -0,0 +1,7 @@
+module github.com/gokrazy/mkfs
```

Comment:
Update this to github.com/gokrazy/mkfs.bcachefs, otherwise `gok add` will add the wrong path to config.json :)